packages/bot-web-ui/src/components/bot-notification-messages/__tests__/bot-notification-messages.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import BotNotificationMessages from '../bot-notification-messages';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
+
+describe('BotNotificationMessages', () => {
+    let wrapper: ({ children }: { children: JSX.Element }) => JSX.Element, mock_DBot_store: RootStore | undefined;
+    beforeAll(() => {
+        const mock_store = mockStore({
+            ui: { notification_messages_ui: jest.fn().mockReturnValue(<p>Mocked Notification Message</p>) as any },
```

Comment:
Just curious why `as any`? @vinu-deriv

Reply:
hey my bad, I forgot to correct it had kept it for later. will change it